src/Features/Core/Portable/UseAutoProperty/AbstractUseAutoPropertyCodeFixProvider.cs

```diff
@@ -85,8 +85,14 @@ private async Task<Solution> ProcessResultAsync(CodeFixContext context, Diagnost
             var project = fieldDocument.Project;
             var compilation = await project.GetRequiredCompilationAsync(cancellationToken).ConfigureAwait(false);
 
+            var renameOptions = new SymbolRenameOptions(
+                RenameOverloads: false,
+                RenameInStrings: false,
+                RenameInComments: false,
+                RenameFile: false);
```

Comment:
curuoyus why explicit values here?

Reply:
this does appear to be the default. so not sure why we're explicit here.

Reply:
No reason. Will remove.